frameworks/kafka/docs/configure.md

```diff
@@ -27,82 +27,53 @@ The Kafka scheduler runs as a Marathon process and can be reconfigured by changi
 1.  Go to the `Services` tab of the DC/OS web interface.
 1.  Click the name of the Kafka service to be updated.
 1.  Within the Kafka instance details view, click the menu in the upper right, then choose **Edit**.
-1.  In the dialog that appears, click the **Environment** tab and update any field(s) to their desired value(s). For example, to [increase the number of Brokers][8], edit the value for `BROKER_COUNT`. Do not edit the value for `FRAMEWORK_NAME` or `BROKER_DISK` or `PLACEMENT_STRATEGY`.
-1.  A `PHASE_STRATEGY` of `STAGE` should also be set. See "Configuration Deployment Strategy" below for more details.
+1.  In the dialog that appears, click the **Environment** tab and update any field(s) to their desired value(s). For example, to [increase the number of Brokers][8], edit the value for `BROKER_COUNT`. Do not edit the value for `FRAMEWORK_NAME` or `BROKER_DISK`.
+1.  Choose a `DEPLOY_STRATEGY`: serial, serial-canary, parallel-canary, or parallel. See the SDK Developer guide for more information on [deployment plan strategies](https://mesosphere.github.io/dcos-commons/developer-guide.html#plans). <!-- I'm not sure I like this solution, since users aren't going to have the context for the dev guide). -->
 1.  Click **REVIEW & RUN** to apply any changes and cleanly reload the Kafka scheduler. The Kafka cluster itself will persist across the change.
 
-## Configuration Deployment Strategy
-
-Configuration updates are rolled out through execution of Update Plans. You can configure the way these plans are executed.
-
-## Configuration Update Plans
-
-In brief, "plans" are composed of "phases," which are in turn composed of "steps." Two possible configuration update strategies specify how the steps are executed. These strategies are specified by setting the `PHASE_STRATEGY` environment variable on the scheduler. By default, the strategy is `INSTALL`, which rolls changes out to one broker at a time with no pauses.
```

Comment:
We can keep this. We may want to edit it little bit.

We have one plan called 'deploy' and we have one phase called 'Deployment'. Those are default names.

Deployment_strategy is actually a phase strategy (see yml file). We should mention it here.